app/migrations/changelogs/notification/MakeParsedDetailsOptional.scala

```diff
@@ -43,9 +43,8 @@ class MakeParsedDetailsOptional extends MigrationDefinition with Logging {
   override val migrationInformation: MigrationInformation =
     MigrationInformation(
       id = "CEDS-2786 Make the parsed detail fields fileReference, outcome and filename optional",
-      order = 1,
-      author = "Tim Wilkins",
-      runAlways = true
+      order = 2,
```

Comment:
You should not alter any existing migration job except maybe to set `runAlways` to false. The order is important to maintain as one job might rely on another being completed before it.